src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -145,6 +147,20 @@ export default {
 			return this.messages[0].systemMessage.length !== 0
 		},
 	},
+
+	methods: {
+		highlightMessage(messageId) {
+			this.$refs.message.every(message => {
+				if (message.id === messageId) {
+					message.highlightMessage()
+					return false
+				} else {
+					// Continue iterating over messages
+					return true
+				}
+			})
+		},
```

Comment:
Semantics of [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) method is to check if every element pass some check and return true/false.

Here it is used as a for loop with implicit break and side effect.

I'd prefer just a loop with return when we need to loop over array and break on some moment.

```js
for (const message of this.$refs.message) {
  if (message.id === messageId) {
    message.highlightMessage()
		return
	}
}
```

Reply:
In vue 3 we can just use ref by id

Reply:
replced `return` with `break` inside the cycle, as we have some post-effects after the loop iterating